ui/components/radio-button-group/base/_index.scss

```diff
@@ -86,16 +99,28 @@
     &:checked ~ .slds-radio--faux,
     &:checked + .slds-radio_button__label,
     &:checked + .slds-radio--button__label {
-      background-color: $color-background-button-brand;
-      color: $color-text-button-brand;
+      background-color: $button-color-background-primary;
+      color: #54698D;
 
       // Need something stronger when the input is checked
       &:hover,
       &:focus {
-        background-color: $color-background-button-brand;
+        background-color: transparent;
       }
     }
 
+    &:checked + .slds-radio_button__label .slds-radio-overlay {
+      position: absolute;
+      border: 2px solid #FF6D00;
+      top: -2px;
+      right: -2px;
+      left: -2px;
+      bottom: -2px;
+      display: block;
```

Comment:
I'm not sure how the top/left/right/bottom values are being used, but you may need to use `rem` values instead of `px`